lib/tuya.js

```diff
@@ -1244,6 +1249,7 @@ const valueConverterBasic = {
 const valueConverter = {
     trueFalse: valueConverterBasic.lookup({1: true, 0: false}),
     onOff: valueConverterBasic.lookup({'ON': true, 'OFF': false}),
+    systemMode: valueConverterBasic.lookup({'heat': false, 'off': true}),
```

Comment:
The values are the opposite here, sorry it was my mistake in the original file
```suggestion
    systemMode: valueConverterBasic.lookup({'heat': true, 'off': false}),
```